tests/findObjectsDifferences.test.js

```diff
@@ -164,4 +164,25 @@ describe('findObjectsDifferences not shallow', () => {
       }
     ])
   })
+
+  test('For sets with same values', () => {
+    const prev = new Set([1, 2, 3])
+    const next = new Set([1, 2, 3])
+    const diffs = findObjectsDifferences(prev, next, {shallow: false})
+    expect(diffs).toEqual([])
+  })
+
+  test('For sets with different values', () => {
+    const prev = new Set([1, 2, 3])
+    const next = new Set([4, 5, 6])
+    const diffs = findObjectsDifferences(prev, next, {shallow: false})
+    expect(diffs).toEqual([
+      {
+        pathString: '',
+        diffType: diffTypes.deepEquals,
```

Comment:
the expected diffType here is "different" since the sets are indeed different.
(different sets are not reported in the console later)

please add a **new test** to catch deep equals changes when:
```
    const prev = new Set([1, 2, 3])
    const next = new Set([1, 2, 3])
```
this is the case where why did you render would indeed tell you nothing changed

Reply:
Ah gotcha - I misunderstood, thanks for that clarification.